src/components/Header/Notifications/Modal/Modal.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheckDouble, faCheck, faTrash } from '@fortawesome/free-solid-svg-icons';
+import './Modal.scss';
+import {
+  getNotification,
+  getUnseenNotification,
+  updateUnseenNotification,
+  deleteNotification
+} from '../../../../actions/notificationActions';
+
+class Modal extends Component {
+  updateStatus = (val) => {
+    const { updateUnseenNotification, getNotification } = this.props;
+    updateUnseenNotification(val);
+    if (val.status !== 'seen') {
+      setTimeout(() => {
+        getNotification();
+      }, 100);
+    }
+  };
+
+  deleteNotificat = (val) => {
+    const { deleteNotification } = this.props;
+    deleteNotification(val);
+  };
+
+  render() {
+    const { notifications, showModal, closeModal } = this.props;
+    return (
+      <div className="container">
+        <div className="card">
+          <div id="modal" className={showModal ? '' : 'hide'}>
+            <form>
```

Comment:
it can be better if you use the common Form

Reply:
Done